docs/admin/kubeadm.md

```diff
@@ -1003,6 +1003,29 @@ In summary, `kubeadm init --feature-gates=SelfHosting=true` works as follows:
 
 This process (steps 3-6) can also be triggered with `kubeadm phase selfhosting convert-from-staticpods`.
 
+## Customising the control plane with custom arguments {#custom-args}
+
+If you would like to override or extend the behaviour of a control plane component, you can provide 
+extra arguments to kubeadm. When the component is deployed, it will use these additional arguments in 
+the pod command itself. 
```

Comment:
it might be a bit clearer to say "...it will use these additional arguments *in its pod command*", but if you disagree I'll defer